EasyModbusSource/LICENSE/LICENSE.txt

```diff
@@ -1,4 +1,4 @@
-Copyright 2018 Vantiq, Inc.
+Copyright 2020 Vantiq, Inc.
```

Comment:
This is ok, but this should really be in a separate PR.  Except for the initial ones, PR's should be relatively small where feasible, addressing a single issue.  Having big PRs that cross connector boundaries is usually not a good idea.

Reply:
ok , you right .